addons/mail/static/src/models/message/message.js

```diff
@@ -554,6 +554,7 @@ function factory(dependencies) {
         checkedThreadCaches: many2many('mail.thread_cache', {
             inverse: 'checkedMessages',
         }),
+        composer: many2one('mail.composer'),
```

Comment:
Add docstring correctly explaining the goal of the field and when it is expected to be set, because this is a bit strange.

Probably it's only used to read some specific values, and those probably should be copied over into the temporary message instead of adding this reference.

Reply:
This relation allow the message component to "retry" (`processMessagesToBeSent` is a composer function) .